src/components/Book.js

```diff
@@ -9,49 +9,20 @@ const Book = ({ book }) => {
   };
 
   return (
-    <div className="book-wrapper">
-      <div className="book">
-        <div className="book-detail">
-          <p className="bk-genre">
-            {' '}
-            {book.category}
-            {' '}
-          </p>
-          <h2 className="bk-title">
-            {' '}
-            {book.title}
-            {' '}
-          </h2>
-          <p className="bk-author">
-            {' '}
-            {book.author}
-            {' '}
-          </p>
-        </div>
-        <div className="bookBtn">
-          <button type="button"> Comments </button>
-          <button type="button" onClick={handleRemoveBook}>
-            Remove
-          </button>
-          <button type="button"> Edit </button>
-        </div>
-      </div>
-
-      <div className="bk-status">
-        <div className="pie-graph"> </div>
-        <div className="completed-percentage">
-          <p className="percentage">64%</p>
-          <p className="completed">Completed</p>
-        </div>
-      </div>
-
-      <div className="process">
-        <div className="curr-chapter">
-          <p className="completed">Current Chapter</p>
-          <p className="chapter">Chapter  17</p>
-        </div>
-        <button type="button">UPDATE PROCESS</button>
-      </div>
+    <div className="book-details">
+      <p className="bk-title">
+        {' '}
+        {book.title}
+        {' '}
+      </p>
+      <p className="bk-author">
+        {' '}
+        {book.author}
+        {' '}
+      </p>
+      <button type="button" onClick={handleRemoveBook}>
+        Remove
+      </button>
     </div>
```

Comment:
- [ ] Please let's try to follow the design guidelines as closely as possible by making the book card look exactly like the Zeplin design
- [ ] Please add the circular progressbar icon
- [ ] Kindly use the appropriate colors for the buttons in the book card
- [ ] Everything that needs to be done is annotated in the image attached below 👇 
## Zeplin Design
![image](https://user-images.githubusercontent.com/46839548/218285134-0b54b210-d8a3-4836-9071-eb121ca84fd6.png)
## Your Design
![image](https://user-images.githubusercontent.com/46839548/218285204-30271ebd-7dd9-42c7-9193-83a831a02b8c.png)